wave_lang/kernel/wave/compile.py

```diff
@@ -94,24 +113,82 @@ def invoke(self, *args, **kwargs):
             arg_idx, dim = self.symbols_args_map[sym]
             dynamic_symbols.append(args[arg_idx].shape[dim])
 
-        invoke_vmfb(
-            self.executable,
-            self.options,
-            kernel_inputs,
-            kernel_outputs,
-            scalar_args,
-            self.bound_scalar_symbols,
-            dynamic_symbols,
-            self.gpu_func,
-        )
+        if self.options.wave_runtime:
+            invoke_with_wave_runtime(
+                self.options,
+                kernel_inputs,
+                kernel_outputs,
+                scalar_args,
+                self.bound_scalar_symbols,
+                dynamic_symbols,
+                self.gpu_func,
+            )
+        else:
+            tensors = [t.data for t in chain(kernel_inputs, kernel_outputs)]
+            self.launchable(*tensors, *scalar_args, *dynamic_symbols)
+
+            if self.options.run_bench:
+                benchmark_flags = {}
+                benchmark_flags["batch_size"] = self.options.benchmark_batch_size
+
```

Comment:
If `self.options.run_bench` is True, we will not actually bench this kernel if `self.options.wave_runtime` is also true, right? 
While we do not have this functionality in wave runtime let's abort or print a warning if both options are true.

Reply:
yeah, I will add a check